src/components/FeeSummary/FeeSummary.vue

```diff
@@ -5,36 +5,46 @@
       :payURL="payApiUrl"
     />
 
-    <v-row class="mt-1">
-      <v-col cols="8" class="pr-2">
+    <v-row no-gutters>
```

Comment:
This looks good.

FYI, I've been going the "no gutters" route as well in Create UI. Typical template:
```
<v-row no-gutters>
  <v-col cols="12" sm="3" class="pr-4">
     My Label
  </v-col>

  <v-col cols="12" sm="9">
    My Data
  </v-col>
</v-row>
```

The `cols="12"` is to make the column full-width under size "sm".

`sm` seems to be a good breakpoint for the Create UI entry and review pages.

The `class="pr-4"` is to add a "gutter" to the label column so the label doesn't run into the data column. (The data column does need this.)

Reply:
Very cool! 
Yes i definitely agree and try to follow this route whenever possible. 
This component was pretty easy.